<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>589</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>随机点名设置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>18</number>
   </property>
   <property name="leftMargin">
    <number>24</number>
   </property>
   <property name="topMargin">
    <number>24</number>
   </property>
   <property name="rightMargin">
    <number>24</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="TitleLabel" name="TitleLabel">
     <property name="text">
      <string>随机点名</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="SmoothScrollArea" name="SmoothScrollArea">
     <property name="styleSheet">
      <string notr="true">background: transparent; border: none</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>640</width>
        <height>508</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="SubtitleLabel" name="SubtitleLabel_2">
           <property name="text">
            <string>名单</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CardWidget" name="CardWidget_3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>70</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="leftMargin">
             <number>16</number>
            </property>
            <property name="topMargin">
             <number>16</number>
            </property>
            <property name="rightMargin">
             <number>16</number>
            </property>
            <property name="bottomMargin">
             <number>16</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="StrongBodyLabel" name="StrongBodyLabel_5">
                <property name="text">
                 <string>编辑名单</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="CaptionLabel" name="CaptionLabel_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>打开名单文件进行编辑</string>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
                <property name="lightColor" stdset="0">
                 <color alpha="150">
                  <red>0</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </property>
                <property name="darkColor" stdset="0">
                 <color alpha="200">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="PrimaryPushButton" name="open_names_list">
              <property name="text">
               <string>打开名单列表</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="SubtitleLabel" name="probSubtitle">
           <property name="text">
            <string>点名概率</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CardWidget" name="probabilityCard">
           <layout class="QVBoxLayout" name="probabilityLayout">
            <property name="leftMargin">
             <number>16</number>
            </property>
            <property name="topMargin">
             <number>16</number>
            </property>
            <property name="rightMargin">
             <number>16</number>
            </property>
            <property name="bottomMargin">
             <number>16</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="StrongBodyLabel" name="StrongBodyLabel_6">
                <property name="text">
                 <string>调整点名概率</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="CaptionLabel" name="CaptionLabel_3">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>设置每位学生被点名的概率等级</string>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="PushButton" name="probability_settings">
              <property name="text">
               <string>设置点名概率</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="SubtitleLabel" name="historySubtitle">
           <property name="text">
            <string>历史记录</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CardWidget" name="historyCard">
           <layout class="QVBoxLayout" name="historyLayout">
            <property name="leftMargin">
             <number>16</number>
            </property>
            <property name="topMargin">
             <number>16</number>
            </property>
            <property name="rightMargin">
             <number>16</number>
            </property>
            <property name="bottomMargin">
             <number>16</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_10">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="StrongBodyLabel" name="StrongBodyLabel_7">
                <property name="text">
                 <string>点名历史</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="CaptionLabel" name="CaptionLabel_4">
                <property name="text">
                 <string>查看最近的点名记录</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="PushButton" name="view_history">
              <property name="text">
               <string>查看点名历史</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimaryPushButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CardWidget</class>
   <extends>QFrame</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SmoothScrollArea</class>
   <extends>QScrollArea</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CaptionLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>StrongBodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SubtitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
